core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -1468,6 +1468,13 @@ private static RelDataType deriveTypeMapConcat(SqlOperatorBinding opBinding) {
           ReturnTypes.TO_MAP_VALUES_NULLABLE,
           OperandTypes.MAP);
 
+  /** The "MAP_CONTAINS_KEY(map, key)" function. */
+  @LibraryOperator(libraries = {SPARK})
+  public static final SqlFunction MAP_CONTAINS_KEYS =
+      SqlBasicFunction.create(SqlKind.MAP_CONTAINS_KEY,
+          ReturnTypes.BOOLEAN_NULLABLE,
+          OperandTypes.MAP_FUNCTION);
```

Comment:
I don't think `OperandTypes.MAP_FUNCTION` is right.

Reply:
Should I write a MapElementOperandTypeChecker class?

Reply:
@caicancai  Do you know map type column?
You only support Map Function.

Reply:
Map, Element？